internal/backend/user_state.go

```diff
@@ -3,6 +3,7 @@ package backend
 import (
 	"context"
 	"fmt"
+
```

Comment:
adding this blank line is good but i wonder why there's also a blank line below `gluon/internal/remote`, that must have snuck in somehow. Mind deleting it?

Reply:
goimports adds a new line here.

Reply:
Yes, as it should. I'm referring to the other blank line:

```go
	"github.com/ProtonMail/gluon/internal/remote"
// this one snuck in somehow... it should be removed
	"github.com/ProtonMail/gluon/internal/backend/ent"
```

Reply:
Fixed